uber/site_sections/preregistration.py

```diff
@@ -215,7 +215,7 @@ def prereg_payment(self, session, payment_id, stripeToken):
         elif charge.amount != charge.total_cost:
             message = 'Our preregistration price has gone up; please fill out the payment form again at the higher price'
         else:
-            message = charge.charge_cc(stripeToken)
+            message = charge.charge_cc(session, stripeToken)
```

Comment:
This is the part of this PR I'm least sure about. I wanted to always record a payment when we charged a credit card, so I definitely wanted that call to be in the `charge_cc` function (where we actually send Stripe the Charge object and tell it to debit someone's card). However, passing that function the session was the only way to prevent the session from going haywire. No other function in the utils class has a session, though. :/